widget/embedded/src/containers/Inputs/Inputs.tsx

```diff
@@ -55,20 +60,21 @@ export function Inputs(props: PropTypes) {
 
   const priceImpactInputCanNotBeComputed = !canComputePriceImpact(
```

Comment:
the second argument of the `canComputePriceImpact` function is the input amount, not the input USD amount.

Reply:
fixed